<?xml version="1.0"?>
<launch>

	<group ns="script_server">
		<!-- upload default parameters to parameter server -->
		<include file="$(find cob_default_config)/robots/$(env ROBOT)/upload_param.launch"/>
		<include file="$(find cob_default_config)/envs/upload_env_param.launch"/>

		<!-- start script_server -->
		<node pkg="cob_script_server" type="script_server.py" name="script_server" cwd="node" respawn="false" output="screen" />
	</group>
	
	<!-- include rostest -->
	<include file="$(find cob_script_server)/launch/script_server.test" />

</launch>
